site2/docs/getting-started-standalone.md

```diff
@@ -58,7 +58,51 @@ Directory | Contains
 `logs` | Logs created by the installation
 
 
+## Installing Builtin Connectors
 
+Since release `2.1.0-incubating`, Pulsar releases a separate binary distribution, containing all the `builtin` connectors.
+If you would like to enable those `builtin` connectors, you can download the connectors tarball release in one of the following ways:
+
+* by clicking the link below and downloading the release from an Apache mirror:
+
+  * <a href="pulsar:connector_release_url" download>Pulsar IO Connectors {{pulsar:version}} release</a>
```

Comment:
You will update the url with the correct link after the binaries are built?

Reply:
pulsar:connector_release_url are templates. so it will be replaced when building the website.

Reply:
gotcha